composer.json

```diff
@@ -176,6 +181,7 @@
         "ext-xmlwriter": "*",
         "bamarni/composer-bin-plugin": "^1.8.2",
         "brianium/paratest": "^7.3",
+        "cweagans/composer-patches": "^1.7",
```

Comment:
this needs to go to require and not require dev right? When we actually go this route